geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/TcpServerFactory.java

```diff
@@ -35,7 +34,7 @@
   public TcpServerFactory() {
     ClientProtocolService service = null;
     try {
-      service = new ClientProtocolServiceLoader().loadService();
+      service = new ClientProtocolServiceLoader().lookupService();
     } catch (ServiceLoadingFailureException ex) {
       logger.warn("Could not load client protocol", ex);
```

Comment:
Is this the line that should change to debug?

Reply:
Update to the PR coming. Just running tests....